R/forecast-bbs-core.R

```diff
@@ -255,18 +261,20 @@ get_pop_ts_env_data <- function(start_yr, end_yr, min_num_yrs){
 get_richness_ts_env_data <- function(start_yr, end_yr, min_num_yrs){
   get_pop_ts_env_data(start_yr, end_yr, min_num_yrs) %>%
```

Comment:
Like this? https://github.com/weecology/bbs-forecasting/pull/64/commits/3da5e8905a1e4d7e6e94be05ae3847329a08d0d6